### ISSUE TITLE
Incorrect date

### ISSUE THREAD

**Issue by canasdiaz** [MetricsGrimoire/eventizer#6]
The date we are storing in the database is not correct, we are using the function datetime.fromtimestamp which is applying the local time so the stored MySQL value is not UTC.

**Comment by jgbarah** [comment]
If you're getting the time as a timestamp, and want to convert it to datetime, you need to use the "aware" version of datetime, which means that you have to set the timezone field in it. See the [datetime doc](https://docs.python.org/2/library/datetime.html).

But have in mind that "Python datetime API always returns unaware datetime objects" (see [Timezones and Python](https://julien.danjou.info/blog/2015/python-and-timezones) ). See this recomendation in that post:

"Always use aware datetime object, i.e. with timezone information. That makes sure you can compare them directly (aware and unaware datetime objects are not comparable) and will return them correctly to users".

In our case, make sure that we store datetime and TZ  in the database, and when reading, compose an aware datetime object, by setting its datetime and TZ.

For example: datetime.datetime(2015, 6, 15, 14, 45, 19, 182703, tzinfo=<UTC>).

In other words, we can store the localtime (local for the time of the event), plus TZ, and then as soon as we recover from the database, produce the aware datetime object, which refernces a certain TZ. This will allow us to do TZ analysis, but at the same time having "aware" times, suitable for comparisons, for example.
